service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/diagnosticreport/ObservationMapperTest.java

```diff
@@ -128,32 +120,31 @@ public void When_MappingObservationJson_Expect_CompoundStatementXmlOutput(String
         String expectedXmlOutput = ResourceTestFileUtils.getFileContent(outputXml);
 
         String compoundStatementXml = observationMapper.mapObservationToCompoundStatement(
-            observationAssociatedWithSpecimen,
-            observations
+            observationAssociatedWithSpecimen
         );
 
         assertThat(compoundStatementXml).isEqualTo(expectedXmlOutput);
     }
 
-    @Test
-    public void When_MappingDefaultObservationJson_Expect_DefaultObservationStatementXmlOutput() throws IOException {
-        when(idMapper.getOrNew(any(ResourceType.class), any(IdType.class))).thenReturn("some-id");
-
-        String jsonInput = ResourceTestFileUtils.getFileContent(
-            OBSERVATION_TEST_FILE_DIRECTORY + "input_default_observation.json"
-        );
-        Observation observationAssociatedWithSpecimen = new FhirParseService().parseResource(jsonInput, Observation.class);
-        String expectedXmlOutput = ResourceTestFileUtils.getFileContent(
-            OBSERVATION_TEST_FILE_DIRECTORY + "expected_output_default_observation.xml"
-        );
-
-        String compoundStatementXml = observationMapper.mapObservationToCompoundStatement(
-            observationAssociatedWithSpecimen,
-            Collections.emptyList()
-        );
-
-        assertThat(compoundStatementXml).isEqualTo(expectedXmlOutput);
-    }
+    // TODO: Fix as part of NIAD-1469
```

Comment:
🚨

Reply:
This wouldn't be forgotten about (as https://gpitbjss.atlassian.net/browse/NIAD-1469 is a sub-task of NIAD-910).
So I thought fixing this test would make more sense for that task, but realised it's easy enough to fix this test as part of this branch.